integration/spring-pulsar/build.gradle

```diff
@@ -0,0 +1,16 @@
+plugins {
+    id 'java'
+    id 'org.springframework.boot'
+    id 'org.springframework.aot.smoke-test'
+    id 'org.graalvm.buildtools.native'
+}
+
+dependencies {
+    implementation(platform(org.springframework.boot.gradle.plugin.SpringBootPlugin.BOM_COORDINATES))
+    implementation("org.springframework.pulsar:spring-pulsar-spring-boot-starter:0.1.0-SNAPSHOT")
```

Comment:
I just noticed @sobychacko that the other modules that specify versions in build.gradle (all the cloud/** ones) are using the `ext { set(` syntax such as: https://github.com/spring-projects/spring-aot-smoke-tests/blob/60a6e70feeed37fadd524cd5fef0ad194a6128a7/cloud/cloud-function-web/build.gradle#L8-L10

Not a big deal, but for consistency it may be a good idea to follow suit here.